includes/AddOns/SteelDesign/UltimateConfigurationEC3.js

```diff
@@ -0,0 +1,330 @@
+ * @param {Boolean} property_determine_lateral_torsion_eq_6_56_or_6_57              If possible, according to Eq. 6.57, otherwise according to Eq. 6.56, can be undefined (is not set, true as default)
+ * @param {Boolean} property_determine_lateral_torsion_f_factor                     Use factor f for modification of chi L, T acc. to 6.3.2.3(2), can be undefined (is not set, true as default)
+ * @param {Boolean} property_perform_design_for_doubly_symmetric_hollow_sections    Perform design for non-circular doubly symmetric hollow sections, can be undefined (is not set, true as default)
+ */
+SteelDesignUltimateConfigurationEC3.prototype.LateralTorsionalBuckling = function (property_determine_lateral_torsion_eq_6_56,
+    property_determine_lateral_torsion_eq_6_56_or_6_57,
+    property_determine_lateral_torsion_f_factor,
+    property_perform_design_for_doubly_symmetric_hollow_sections) {
+    ASSERT(this.addon.settings_ec3.property_perform_stability_analysis, "Perform stability design must be on");
+    if (typeof property_determine_lateral_torsion_eq_6_56 !== "undefined") {
+        this.addon.settings_ec3.property_determine_lateral_torsion_eq_6_56 = property_determine_lateral_torsion_eq_6_56;
+    }
+    if (typeof property_determine_lateral_torsion_eq_6_56_or_6_57 !== "undefined") {
+        this.addon.settings_ec3.property_determine_lateral_torsion_eq_6_56_or_6_57 = property_determine_lateral_torsion_eq_6_56_or_6_57;
+    }
+    if (typeof property_determine_lateral_torsion_f_factor !== "undefined") {
+        this.addon.settings_ec3.property_determine_lateral_torsion_f_factor = property_determine_lateral_torsion_f_factor;
+    }
+    if (typeof property_perform_design_for_doubly_symmetric_hollow_sections !== "undefined") {
+        this.addon.settings_ec3.property_perform_design_for_doubly_symmetric_hollow_sections = property_perform_design_for_doubly_symmetric_hollow_sections;
+    }
+};
+
+/**
+ * Sets Parameters
+ * @param {Boolean} property_param_k_annex_a    Method 1 acc. to Annex A, can be undefined (is not set, false as default)
+ * @param {Boolean} property_param_k_annex_b    Method 2 acc. to Annex B, can be undefined (is not set, true as default)
+ */
+SteelDesignUltimateConfigurationEC3.prototype.Parameters = function (property_param_k_annex_a,
+    property_param_k_annex_b) {
+    ASSERT(this.addon.settings_ec3.property_perform_stability_analysis, "Perform stability design must be on");
+    if (typeof property_param_k_annex_a !== "undefined") {
+        this.addon.settings_ec3.property_param_k_annex_a = property_param_k_annex_a;
+    }
+    if (typeof property_param_k_annex_b !== "undefined") {
+        this.addon.settings_ec3.property_param_k_annex_b = property_param_k_annex_b;
+    }
+}
+
+SteelDesignUltimateConfigurationEC3.prototype.GeneralRules = function () {
+    ASSERT(false, "GeneralRules, no API support?");
+};
+
+SteelDesignUltimateConfigurationEC3.prototype.ColdFormDesign = function() {
```

Comment:
Why this function if it only throws assert?

Reply:
Deleted. Only comment above.